open_lm/file_utils.py

```diff
@@ -285,12 +289,15 @@ def get_string_for_epoch(
     num_workers_per_gpu: int,
     world_size: int,
     multi_epoch=False,
+    shard_shuffle_seed=None,
 ):
     """See _single_epoch_string for full docstring."""
     if multi_epoch:
         raise NotImplementedError("Multiple passes over the dataset not fully supported yet.")
     else:
-        return _single_epoch_string(num_samples, starting_points, paths, weights, num_workers_per_gpu, world_size)
+        return _single_epoch_string(
+            num_samples, starting_points, paths, weights, num_workers_per_gpu, world_size, shard_shuffle_seed
+        )
 
 
 def _multi_epoch_string(num_samples, starting_chunk, paths, weights, min_shards_needed):
```

Comment:
just curious what this function is

Reply:
seems not used, but just want to understand a bit better

Reply:
It is unused for now, but it is part of a codepath for multi epoch, which will be implemented later - it currently cannot be accessed.

Reply:
I just made a wrapper between both cases here.